src/Controls/src/Build.Tasks/Controls.Build.Tasks.csproj

```diff
@@ -10,52 +10,79 @@
+			<DefineConstants Condition=" '$(_DisableStrongNamer)' != 'True' ">$(DefineConstants);STRONG_NAME</DefineConstants>
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Mono.Cecil" Version="0.11.3" />
-		<PackageReference Include="System.CodeDom" Version="5.0.0" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
-		<PackageReference Include="Microsoft.Build" Version="15.8.166" />
-		<PackageReference Include="Microsoft.Build.Framework" Version="15.8.166" />
-		<PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.8.166" />
-		<PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.8.166" />
-		<PackageReference Include="StrongNamer" Version="0.2.5" />
-		<PackageReference Include="System.ValueTuple" Version="4.5.0" />
+		<PackageReference Include="Mono.Cecil" Version="0.11.3" PrivateAssets="all" GeneratePathProperty="true" />
+		<PackageReference Include="System.CodeDom" Version="5.0.0" PrivateAssets="all" GeneratePathProperty="true" />
+		<PackageReference Include="Microsoft.Build" Version="15.9.20" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.Build.Framework" Version="15.9.20" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.9.20" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.9.20" PrivateAssets="all" />
+		<PackageReference Include="StrongNamer" Version="0.2.5" PrivateAssets="all" GeneratePathProperty="true" />
 		<PackageReference Include="XliffTasks" Version="1.0.0-beta.21110.1" PrivateAssets="all" />
 	</ItemGroup>
 
+	<PropertyGroup>
+		<PackageId>Microsoft.Maui.Controls.Build.Tasks</PackageId>
+		<IncludeBuildOutput>False</IncludeBuildOutput>
+		<NoWarn>$(NoWarn);NU5100;NU5128;</NoWarn>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<None Include="$(TargetPath)" Visible="False" Pack="True" PackagePath="buildTransitive;build" />
+
+
+		<None Include="..\..\..\..\.nuspec\Microsoft.Maui.Controls.props" Visible="False" Pack="True" PackagePath="buildTransitive;build" />
+		<None Include="..\..\..\..\.nuspec\Microsoft.Maui.Controls.targets" Visible="False" Pack="True" PackagePath="buildTransitive;build" />
+		<None Include="..\..\..\..\.nuspec\Microsoft.Maui.Controls.SingleProject.props" Visible="False" Pack="True" PackagePath="buildTransitive;build" />
+		<None Include="..\..\..\..\.nuspec\Microsoft.Maui.Controls.SingleProject.targets" Visible="False" Pack="True" PackagePath="buildTransitive;build" />
+
+		<None Include="..\..\..\..\.nuspec\Microsoft.Maui.Controls.DefaultItems.props" Visible="False" Pack="True" PackagePath="buildTransitive\Microsoft.Maui.Controls.Build.Tasks.props;build\Microsoft.Maui.Controls.Build.Tasks.props" />
+		<None Include="..\..\..\..\.nuspec\Microsoft.Maui.Controls.DefaultItems.targets" Visible="False" Pack="True" PackagePath="buildTransitive\Microsoft.Maui.Controls.Build.Tasks.targets;build\Microsoft.Maui.Controls.Build.Tasks.targets" />
```

Comment:
Maybe we should rename the file instead . Or, is this meant to be here at all? Shouldn't this be in MS.Maui.Core?